docker-compose.yml

```diff
@@ -134,6 +141,19 @@ services:
     volumes:
       - openwisp_certs:/etc/ssl/mail
 
+  freeradius:
+    image: openwisp/openwisp-freeradius:latest
+    build:
+      context: build
+      dockerfile: openwisp_freeradius/Dockerfile
+    deploy:
+      replicas: 1
+    env_file:
+      - .env
+    depends_on:
+      - postgres
+      - dashboard
```

Comment:
I think this should be radius container.

Reply:
As per my understanding,since both the radius and dashboard container have the radius API, so I think freeradius container should work with both dashboard and radius containers. The default one that I am testing on is using dashboard, hence I have put dashboard here.

If a user is creating a new instance of openwisp-radius container, they'll have to create a freeradius container working with it. (I think I should add this in the documentation.)
However, since the openwisp-radius container is optional while openwisp-dashboard is always going to be there, I thought that I should keep dashboard as default here. 

Please let me know your thoughts on this. :smile:

Reply:
I thought the purpose of the openwisp-radius container was for serving the Radius API.  I didn't think we were going to leave the Radius API urls as part of dashboard.  I may have misunderstood so we should at least have documentation that makes it less confusing.

Reply:
APIs in general should not be part of the dashboard

Reply:
Got it, thanks, opening a seperate issue to remove API from dashboard container. #52